zuri_net/src/client/data.rs

```diff
@@ -106,6 +106,8 @@ pub struct ClientData {
     pub ui_profile: i32,
 
     pub trusted_skin: bool,
+    /// The usage of this field is currently unknown.
```

Comment:
~~The player will refuse to use any skin the server sends that is different to their client-side skin unless this field is set~~ Just realised this is in the client data. Not 100% sure why it is in both places but I'm guessing it's like a master key

Reply:
I imagine they use the same `Skin` type for player_list/player_skin as here